lib/Utils/Logging.pm

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Save logs directly on the worker for offline upload via ulogs
+# see https://progress.opensuse.org/issues/109792
+
+package Utils::Logging;
+
+use base 'Exporter';
+use Exporter;
+use strict;
+use warnings;
+use testapi 'script_output';
+use Mojo::File qw(path);
+
+our @EXPORT = qw(log2serial);
+
+sub log2serial {
+    my $data = script_output("journalctl -k | gzip | base64 -w0");
+    my $logname = "temporary-filename.log.gz";
```

Comment:
since the upload of the logs works out of the box, modify logs2serial, so that it recieves 2 parameters, one passing the content of the file, and a second one passing the filename...

and let's call it here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f34f7221e0bc2f44c2202ab1c851b6b7ae1936e9/tests/console/start_systemd_testkit_offline.pm#L123

so that these logs, are uploaded here: https://openqa.suse.de/tests/9247366#step/start_systemd_testkit_offline/60